src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,144 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    // final int hash - it's a Object.hashCode() of key
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final int MAXIMUM_CAPACITY = 1 << 30;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K,V>[] table;
+    private int threshold;
+    private int size;
+
+    public MyHashMap() {
+        this.table = (Node<K,V>[])new Node[DEFAULT_INITIAL_CAPACITY];
+        this.threshold = (int) (DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR);
+        this.size = 0;
```

Comment:
```suggestion
```
int field has 0 by default

Reply:
done

Reply:
Not done
![image](https://user-images.githubusercontent.com/20326858/216828482-232a4d8b-7547-479d-9bba-a6b9ce6ee6cd.png)